packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte

```diff
@@ -14,6 +14,9 @@
 	export let row = 0;
 	export let column = null;
 
+	//Passing in link for anchor tag
+	export let link;
```

Comment:
```suggestion
	export let link = undefined;
```
prevents console warnings

Reply:
nice, will update